furusiya/model/key_binder.py

```diff
@@ -0,0 +1,85 @@
+from model.keys.events import update_callback, quit_event, mousemotion_event
+from model.keys.key_callbacks import *
+
+
+keybinds = {
+    'ESCAPE': exit_to_main_menu_callback,
+    'ENTER': enter_callback,
+
+    'UP': up_callback,
+    'DOWN': down_callback,
+    'LEFT': left_callback,
+    'RIGHT': right_callback,
+
+    'g': pickup_callback,
+
+    'i': inventory_use,
+    'd': inventory_drop,
+
+    'f': bow_callback,
+    'm': mount_callback,
+
+    'r': rest_callback,
+    'R': continuous_rest_callback
+}
+
+events = {
+    'QUIT': quit_event,
+
+    'MOUSEMOTION': mousemotion_event,
+    'MOUSEDOWN': None,
+    'MOUSEUP': None,
+
+    'KEYDOWN': None,
+    'KEYUP': None,
+}
+
+
+class KeyBinder:
```

Comment:
Can we get rid of code that we're not using and probably won't need? Or is all of this new code in-use?